src/features/items.ts

```diff
@@ -181,3 +147,94 @@ export function itemsReducer(
 export const ItemsStore = configureStore({
   reducer: itemsReducer,
 });
+
+export const addItem = (uid: string, url: string) => (dispatch: any) => {
+  dispatch(reload(true));
```

Comment:
It would be nice if that action can get rid of that boolean parameter cause it might be a [bit smelly](https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3).